platform/openide.awt/src/org/openide/awt/SplittedPanel.java

```diff
@@ -779,10 +779,10 @@ private void initAccessible() {
         getAccessibleContext().setAccessibleName(
             nameFormat.format(
                 new Object[] {
-                    ((firstComponent == null) || !(firstComponent instanceof Accessible)) ? null
+                    (!(firstComponent instanceof Accessible)) ? null
                                                                                           : firstComponent.getAccessibleContext()
                                                                                                           .getAccessibleName(),
```

Comment:
could you adjust the formatting here a little bit? this could be one line now.
```java
(!(firstComponent instanceof Accessible)) ? null : firstComponent.getAccessibleContext().getAccessibleName(),
```